src/main/java/com/epam/university/java/core/task033/LessExceptionImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.university.java.core.task033;
+
+class LessExceptionImpl extends BaseExceptionImpl implements LessException {
+    LessExceptionImpl() {
```

Comment:
Don't forget to add access modifier, constructors should be public.

Reply:
Done